exec/java-exec/src/test/resources/testframework/testCastFunctions/testToDateForTimeStamp.tsv

```diff
@@ -0,0 +1 @@
+1969-12-31
```

Comment:
Lets not add a file for a single value, you can probably use the baseLineValues() method.